src/main/java/com/exactpro/th2/common/event/Event.java

```diff
@@ -308,12 +308,12 @@ public com.exactpro.th2.common.grpc.Event toProto(@NotNull String bookName) thro
 
     public com.exactpro.th2.common.grpc.Event toProto(
             @NotNull String bookName,
-            @NotNull String scope
+            @Nullable String scope
```

Comment:
Why scope can be nullable?

Reply:
This method call overload which one use default scope value when passed scope is null